### PR TITLE
[various] Replace remaining $a->page by DI::page()

### PR DESCRIPTION
**Pull request by MrPetovan**
Follow-up to #935
Part of https://github.com/friendica/friendica/pull/8035
Addresses https://github.com/friendica/friendica/issues/8000#issuecomment-569770878

### CODE REVIEW

**Comment by nupplaphil** [comment]
I think I found why I missed this much of changes .. I added another PR and after the push, I missed half of my changes in the remote branch .. Something is really wrong with my IDE ..

**Comment by MrPetovan** [comment]
Does your IDE also prevented you from merging this PR?

**Comment by nupplaphil** [comment]
Not the IDE, just lack of permissions ;-)

**Comment by MrPetovan** [comment]
Oooh, in friendica-addons, right.

File: fromapp/fromapp.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ function fromapp_settings(&$a, &$s)
 
 	/* Add our stylesheet to the page so we can make our settings look nice */
 
-	$a->page['htmlhead'] .= '<link rel="stylesheet"  type="text/css" href="' . DI::baseUrl()->get() . '/addon/fromapp/fromapp.css' . '" media="all" />' . "\r\n";
+	DI::page()['htmlhead'] .= '<link rel="stylesheet"  type="text/css" href="' . DI::baseUrl()->get() . '/addon/fromapp/fromapp.css' . '" media="all" />' . "\r\n";
```
Comment (annando) [inline fromapp/fromapp.php]: I never saw this construct. How is it supposed to work?

Comment (nupplaphil) [reply fromapp/fromapp.php]: It's like:
```
$page = DI::page();
$page['htmlhead'] =  '...';
```

The `Friendica\App\Page` class implements `ArrayAccess` interface, that's why it works like an array. We could also explicit an add method :)

Comment (annando) [reply fromapp/fromapp.php]: I guess I would prefer some "get", "set" and "add" functions.

Comment (nupplaphil) [reply fromapp/fromapp.php]: Me too. Now its possible :)

**Pull request merged**